.github/workflows/docker-validate.yml

```
@@ -0,0 +1,192 @@
+
+on:
+  push:
+    branches: [ main, master ]
+    paths:
+      - 'Dockerfile'
+      - 'docker-compose.yml'
+      - '.dockerignore'
+      - '.github/workflows/docker-validate.yml'
+      - 'prisma/schema.prisma'
+  pull_request:
+    branches: [ main, master ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)